flang/test/Driver/isysroot.f90

```diff
@@ -6,7 +6,7 @@
 ! RUN: %flang -### --target=aarch64-linux-gnu -isysroot /path/to/sysroot \
 ! RUN:        %s 2>&1 | FileCheck %s --check-prefix=CHECK-LINUX
 
-! CHECK-DARWIN: "{{.*[\/]}}ld" {{.*}}"-syslibroot" "/path/to/sysroot"
+! CHECK-DARWIN: "{{.*[\/]}}ld{{(64)?(\.lld)?}}" {{.*}}"-syslibroot" "/path/to/sysroot"
```

Comment:
There is a Windows buildbot that uses "ld" only in the linker command line: https://lab.llvm.org/buildbot/#/builders/120/builds/6174.
I guess it's better to remove the `[\/]` part, or make it optional.

Reply:
OK. I will remove `[\/]`. Thanks for catching that.